### PR TITLE
Fix jsp-file handling

### LINKED ISSUE #1959: jsp-file broken
The `<jsp-file>` element inside a servlet element in web.xml is broken. It used to work.

### PR DESCRIPTION
**Pull request by folker-kuhn**
Fixes #1959

This will fix the jsp-file handling. It moved the wasp extension to right location and removed code that was not needed.

### CODE REVIEW

**Comment by mnriem** [comment]
See #1959

**Review by Thihup — APPROVED** [review]
LGTM

File: extension/server/src/main/java/cloud/piranha/extension/server/ServerExtension.java [hunk 1]
```diff
@@ -48,15 +48,12 @@
     public void extend(WebApplicationExtensionContext context) {
         context.add(HerringExtension.class);
         context.add(AnnotationScanExtension.class);
+        // WaspExtension must be added before WebXmlExtension, so webApplication.addJspFile that's called by WebXmlExtension
```
Comment (mnriem) [inline extension/server/src/main/java/cloud/piranha/extension/server/ServerExtension.java]: Now that you have solved the ordering problem can you remove the comment?

**Review by mnriem — CHANGES_REQUESTED** [review]
Just remove the one comment and then you are good to go. Thank YOU!

**Review by mnriem — APPROVED** [review]

**Comment by mnriem** [comment]
@folker-kuhn Thank you for this contribution!

**Pull request merged**